README.md

```diff
@@ -43,42 +44,17 @@ python -m pip install -r requirements-dev.txt
 pre-commit install
 ```
 
-### Required environment variables
-The Django settings.py file for this project requires setting an environment
-variable: `SECRET_KEY`
-
-Running the following in your shell should print a secret key that can be used.
-```shell
-python
-import secrets
-secrets.token_urlsafe()
-exit()
-
-```
-
-Set the environment variable using *the entire output* (including quotes) from
-the printed secret
-```shell
-# BASH-like shells
-export SECRET_KEY=<your-secret-here>
-```
-```powershell
-# PowerShell
-$Env:SECRET_KEY=<your-secret-here>
-```
-Note: during development, it may also be helpful to add the `DEBUG` environment
-variable and setting it to the string `True`
-
 ### Running the application
-After completing [development setup](#development-setup) and
-[environment variable setup](#required-environment-variables) you can run the
-application locally with:
+After completing [development setup](#development-setup) application locally with:
 ```shell
-python manage.py collectstatic
-python manage.py test --debug-mode
-gunicorn -b 127.0.0.1:8080 usps.wsgi
+pytest # Unit tests should be passing
+uvicorn usps.main:app
 ```
 
+### Viewing API Endpoints and documentation
+Documentation can be viewed locally by running the application and visiting
+http://127.0.0.1:8000/docs
```

Comment:
This points to the swagger doc. Do we want to show off the redoc instead? That would be http://127.0.0.1:8000/redoc

Reply:
Yes. I agree the redoc docs look a bit nicer.